app/controllers/receipts_controller.rb

```diff
@@ -0,0 +1,69 @@
+class ReceiptsController < ApplicationController
+  include CartsHelper
+  before_action :store_location, :logged_in_user
+  before_action :load_cart, :load_room, only: :create
+
+  def index
+    @receipts = Receipt.of_current_user current_user
```

Comment:
Phân trang bạn ơi
cơ mà chỗ ni cần j scope cậu, tự rối mình

```suggestion
    @receipts = @current_user.receipts
```